json/targeting.json

```
@@ -0,0 +1,553 @@
+        },
+        ">=": {
+          "title": "Greater-Than-Or-Equal-To Operation",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "%": {
+          "title": "Modulo Operation",
+          "description": "Finds the remainder after the first argument is divided by the second argument.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "map": {
+          "title": "Map Operation",
+          "description": "Perform an action on every member of an array. Note, that inside the logic being used to map, var operations are relative to the array element being worked on.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "reduce": {
+          "title": "Reduce Operation",
+          "description": "Combine all the elements in an array into a single value, like adding up a list of numbers. Note, that inside the logic being used to reduce, var operations only have access to an object with a \"current\" and a \"accumulator\".",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "filter": {
+          "title": "Filter Operation",
+          "description": "Keep only elements of the array that pass a test. Note, that inside the logic being used to filter, var operations are relative to the array element being worked on.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "all": {
+          "title": "All Operation",
+          "description": "Perform a test on each member of that array, returning true if all pass. Inside the test code, var operations are relative to the array element being tested.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "none": {
+          "title": "None Operation",
+          "description": "Perform a test on each member of that array, returning true if none pass. Inside the test code, var operations are relative to the array element being tested.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "some": {
+          "title": "Some Operation",
+          "description": "Perform a test on each member of that array, returning true if some pass. Inside the test code, var operations are relative to the array element being tested.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "in": {
+          "title": "In Operation",
+          "description": "If the second argument is an array, tests that the first argument is a member of the array.",
+          "$ref": "#/$defs/binaryOp"
+        },
+        "substr": {
+          "title": "Substring Operation",
+          "description": "Get a portion of a string, given a positive start position as an index (indexes of course start at zero).",
+          "$ref": "#/$defs/binaryOp"
+        }
+      }
+    },
+    "associativeOp": {
+      "type": "array",
+      "minItems": 2,
+      "items": {
+        "$ref": "#/$defs/args"
+      }
+    },
+    "associativeRule": {
+      "title": "Mathematically Associative Operation",
+      "description": "Pperation applicable to 2 or more parameters.",
```

```suggestion
      "description": "Operation applicable to 2 or more parameters.",
```

I'm not 100% sure this is what you intended.